### PR TITLE
Add ComposeTransforms

### PR DESCRIPTION
**Pull request by Yancey1989**
related issue #147 
This PR added a `ComposeTransform` to execute GoTorch transforms sequentially.

### CODE REVIEW

File: vision/transforms/transforms.go [hunk 1]
```diff
@@ -13,3 +18,53 @@ type NormalizeTransformer struct {
 func Normalize(mean float64, stddev float64) *NormalizeTransformer {
 	return &NormalizeTransformer{mean, stddev}
 }
+
```
Comment (typhoonzero) [inline vision/transforms/transforms.go:21]: Maybe put `Normalize` function in a seperated file, and `transforms.go` contains only interfaces.

File: vision/transforms/transforms.go [hunk 2]
```diff
@@ -13,3 +18,52 @@ type NormalizeTransformer struct {
...
+// ComposeTransforms composes transforms together
+type ComposeTransforms struct {
```
Comment (wangkuiyi) [inline vision/transforms/transforms.go:23]: Compose is a **transformer**, so the name should be `ComposeTransformer`?

Comment (Yancey1989) [reply vision/transforms/transforms.go:23]: Agree with that, will rename it in the next PR.

File: vision/transforms/transforms.go (hunk 1 above)
Comment (Yancey1989) [reply vision/transforms/transforms.go:21]: I'm not sure, the code of `Normalize` is short, and other functions of Tensor e.g. `Add/Mul/...` are all packaged in `tensor.go`, maybe we can separate it from `transforms.go` if we have many transformers?

**Pull request merged**